OnServiceChange/handler.ts

```diff
@@ -178,16 +179,42 @@ export const log = (d: unknown): void => {
   throw new Error(`To be implement ${d}`);
 };
 
-export const onInvalidDocument = (d: unknown): T.Task<void> => {
+export const onInvalidDocument = (
+  d: unknown,
+  telemetryClient: ReturnType<typeof initTelemetryClient>
+): T.Task<void> => {
+  telemetryClient.trackEvent({
+    name: "selfcare.services.oninvaliddocument",
```

Comment:
Consider using `selfcare.subsmigrations.` as _namespace_

Reply:
Yes, fix it with 61d4cce